app/src/main/java/com/vultisig/wallet/ui/screens/keysign/Keysign.kt

```diff
@@ -43,6 +49,7 @@ internal fun Keysign(
     LaunchedEffect(state) {
         if (state == KeysignState.KeysignFinished) {
             onKeysignFinished?.invoke()
+            reviewManager.showReviewPopUp(context)
```

Comment:
I can can see your consern, but this bottom sheet will be triggered only 3 times if app is not alredy rated. 
And it will be triggered not one after another but with some delay like days